### ISSUE TITLE
Add a tkn cli path settings

### ISSUE THREAD

**Issue by chmouel** [redhat-developer/vscode-tekton#679]
I have tkn installed (actually compiled from source) that is not in my PATH it would be nice if I can be able to specify the tkn cli path instead of having the extension asking me every time to download it...

**Comment by mohitsuman** [comment]
The scenario here is the extension bundled with specific version of `tkn` allows to work with features specific to that release. If users try to run tkn from path, and if the version is very old, this can create compatibility issues for any features and can cause confusion to the workflow.

**Comment by chmouel** [comment]
I understand but maybe offer that choice or just avoid it ? I mostly use the extension for syntax highlight

(and we try to keep tekton compatibility as much as possible on tkn or that would be a bug)

**Comment by mohitsuman** [comment]
Yeah we can have a feature to ask user to provide the path of the installed binary (if extension does not detect it). Is there a version, upto what backward compatibility is supported?

Let's say we have latest tkn as `v0.23.1`, upto which versions of `tkn` would we support backward compatibility. This can help us to notify users in IDE if the path provided is very old.
